asciidocapi.py

```diff
@@ -189,16 +192,23 @@ def __init__(self, asciidoc_py=None):
             if not os.path.isfile(cmd):
                 raise AsciiDocError('missing file: %s' % cmd)
         else:
-            # Try shell search paths.
+            # try to find sibling
+            this_path = os.path.dirname(os.path.realpath(__file__))
+            # Try sibling paths.
```

Comment:
I'd replace the comment two lines up with this one and just have one comment describing this block. Having these both with a line between them is unnecessary as they duplicate the content.